config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,66 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - sriov
+  - networking
+  links:
+  - name: Network Operator Documentation
+    url: https://docs.nvidia.com/networking/display/COKAN10/Network+Operator
+  - name: GitHub repository
+    url: https://github.com/Mellanox/network-operator/
+  maintainers:
+  - email: nvidia-network-operatorsupport@nvidia.com
```

Comment:
@e0ne Is it OK to have this mail here?

Reply:
no, it should be: nvidia-network-operator-support@nvidia.com

Reply:
Fixed